cmd/kueue/main.go

```diff
@@ -217,6 +217,10 @@ func main() {
 	}
 	if cfg.FairSharing != nil {
 		cacheOptions = append(cacheOptions, cache.WithFairSharing(cfg.FairSharing.Enable))
+		queueOptions = append(queueOptions, queue.WithFairSharing(cfg.FairSharing))
```

Comment:
is this used?

Reply:
Good point, after changes to the configuration API this is no longer used, will fix

Reply:
Done